CONTRIBUTING.md

```diff
@@ -2,15 +2,216 @@ Guidelines for Contributing
 ===========================
 
 Welcome to the D community and thanks for your interest in contributing!
-If this is your first pull request, please read [Contributing to Phobos](https://wiki.dlang.org/Contributing_to_Phobos).
-Have a look at [Starting as a Contributor](http://wiki.dlang.org/Starting_as_a_Contributor#Building_D) for build instructions.
-
-More Links
-----------
-
-* Fork [on Github](https://github.com/dlang/phobos)
-* Use our [Bugzilla bug tracker](https://issues.dlang.org)
-* Follow the [D style](http://dlang.org/dstyle.html)
-* Participate in [our forum](http://forum.dlang.org/)
-* Ask questions on our `#d` IRC channel on freenode.org ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
-* Review Phobos additions in the [Review Queue](http://wiki.dlang.org/Review_Queue).
+If you need help you can ask questions on `#d` IRC channel on
+freenode.org ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
+or on [our forum](http://forum.dlang.org/).
```

Comment:
I would link to the [Phobos](https://forum.dlang.org/group/phobos) subsection of the forum.
However, it is currently cluttered by dupes of github notifications. DMD moved away from that a couple of months ago, and druntime and phobos probably should as well. What do you think @braddr ?